### ISSUE TITLE
Model with MobileFacenet backbone + MagFace head

### ISSUE THREAD

**Issue by huytuong010101fpt** [JDAI-CV/FaceX-Zoo#86]
Perfect repository <3
I see that you implemented the MagFace head in your repository 
Can you share with me the pre-trained model which uses MobileFace backbone and MagFace head
I really need that model
Thank you so much <3

**Comment by wang21jun** [comment]
https://drive.google.com/drive/folders/1yJZvfYAE3wmoBXdAIHHAOUXvX0uNaDcM?usp=sharing

**Comment by huytuong010101** [comment]
> https://drive.google.com/drive/folders/1yJZvfYAE3wmoBXdAIHHAOUXvX0uNaDcM?usp=sharing

Sorry, do you have pre-trained model which uses MobileFaceNet backbone and MagFace head. Think the model you giving is not using MobileFaceNet

**Comment by huytuong010101** [comment]
I see the MobileFaceNet pretrained in `FaceX-Zoo/face_sdk/models/face_recognition` but I don't known what `Head` does it use?

**Comment by tmhieu99** [comment]
Hi @huytuong010101, according to the params in the log file, I think the model in google drive is probably MobileFaceNet with MagFace head. And the model in face sdk is trained with the mv-softmax head.

**Comment by qiu-pinggaizi** [comment]
While we don't have the exact MobileFaceNet + MagFace combination, we have a similar lightweight MobileFaceNet model (mbf_v3, 3.7M params) trained on Glint360K with ArcFace loss, and a variant with SE attention (mbf_v3_se, 3.9M params) that achieves:

- LFW: 99.83%
- CFP-FP: 98.71%
- AgeDB-30: 98.05%

We also have a knowledge distillation pipeline that trains a lightweight student (MobileFaceNet) under a frozen teacher (ResNet-100), which is conceptually similar to combining different loss heads. The distillation uses cosine similarity on L2-normalized embeddings.

If you want to try MagFace head with MobileFaceNet backbone, you could adapt our training pipeline by replacing the ArcFace head with MagFace. Our architecture separates backbone (mbf_v3_se) from the classification head (PartialFC), making it straightforward to swap.

Project: https://github.com/qiu-pinggaizi/arcface_torch
